src/core/load-dropin.c

```diff
@@ -19,51 +19,245 @@
 
 
 #include "conf-parser.h"
+#include "fs-util.h"
 #include "load-dropin.h"
 #include "load-fragment.h"
 #include "log.h"
+#include "path-util.h"
+#include "stat-util.h"
 #include "strv.h"
 #include "unit-name.h"
 #include "unit.h"
 
+
+typedef struct DependencyConsumerData {
+        Unit *unit;
+        int depth;
+} DependencyConsumerData;
+
+typedef struct DropinSymlink {
```

Comment:
so far we used the capitalization "DropIn", see the DropInPaths= bus object property...